nipy/modalities/fmri/experimental_paradigm.py

```diff
@@ -48,13 +48,7 @@ def __init__(self, con_id=None, onset=None, amplitude=None):
         self.amplitude = amplitude
         if con_id is not None:
             self.n_events = len(con_id)
-            try:
```

Comment:
I think that in general it is a bad idea to decide that the conditions cannot be named '1', '2' etc. and to change this for the user. Now, if you think that backward compatibility is more important, I can revert that one.